diff -ru rpcs3_orig/3rdparty/libpng/CMakeLists.txt rpcs3/3rdparty/libpng/CMakeLists.txt
--- rpcs3_orig/3rdparty/libpng/CMakeLists.txt	2020-01-07 18:29:55.047688687 +0100
+++ rpcs3/3rdparty/libpng/CMakeLists.txt	2020-01-07 20:20:42.817499344 +0100
@@ -24,6 +24,8 @@
 project(libpng C ASM)
 enable_testing()
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
 set(PNGLIB_MAJOR 1)
 set(PNGLIB_MINOR 6)
 set(PNGLIB_RELEASE 36)
